server/src/main/java/io/druid/guice/DruidProcessingModule.java

```diff
@@ -69,7 +69,7 @@ public ExecutorService getBackgroundExecutorService(
           cacheConfig.getNumBackgroundThreads(),
           new ThreadFactoryBuilder()
               .setNameFormat("background-cacher-%d")
-              .setDaemon(true)
+              .setDaemon(true) // TODO: migrate this to Execs after https://github.com/druid-io/druid/pull/984
```

Comment:
instead of a TODO, we should either used Execs or make a note in the corresponding PR.